client/my-sites/stats/stats-notices/index.tsx

```diff
@@ -106,6 +109,13 @@ const NewStatsNotices = ( { siteId, isOdysseyStats, statsPurchaseSuccess }: Stat
 		hasSiteProductJetpackStatsPWYWOnly( state, siteId )
 	);
 
+	// Determine if the site has the Dotcom upsell visible.
+	const hasWpcomUpsell = useSelector(
+		( state ) =>
+			config.isEnabled( 'stats/paid-wpcom-v3' ) &&
```

Comment:
The feature flag `stats/paid-wpcom-v3` has been flipped so we may not need this check.

Reply:
I did a quick search and the flag is still present in a lot of source files so I'll leave it in for now. I think it makes sense to remove the check when removing the feature flag as it keeps the `git` history tidy.